mmv1/products/privateca/terraform.yaml

```diff
@@ -54,6 +54,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           kms_key_name: 'BootstrapKMSKeyWithPurposeInLocation(t, "ASYMMETRIC_SIGN", "us-central1").CryptoKey.Name'
           pool_name: 'BootstrapSharedCaPoolInLocation(t, "us-central1")'
           pool_location: "\"us-central1\""
+    virtual_fields:
+      - !ruby/object:Api::Type::Boolean
+        name: 'include_max_path_length'
+        default_value: false
+        description: |
+          Must be set to use the value of max_issuer_path_length, either 0 or greater than 0.
+          If not set, max_issuer_path_length will be unset.
```

Comment:
From the description, field `include_max_path_length` seems like an integer (`either 0 or greater than 0`) while its type is Boolean. Is that intended?

Reply:
`include_max_path_length` is meant to be a boolean, whereas `max_issuer_path_length` is an integer. We rely on `include_max_path_length` to differentiate between an unset `max_issuer_path_length` and `max_issuer_path_length` set to a default value of 0, since the API takes an optional integer for `max_issuer_path_length.

Reply:
IC. Thanks